data/rulesets.ts

```diff
@@ -1510,4 +1510,51 @@ export const Rulesets: {[k: string]: FormatData} = {
 			pokemon.trapped = true;
 		},
 	},
+	chimera1v1rule: {
+		effectType: 'Rule',
+		name: 'Chimera 1v1 Rule',
+		desc: "Validation and battle effects for Chimera 1v1.",
+		ruleset: ['Team Preview', 'Max Team Size = 6', 'Min Team Size = 6', 'Picked Team Size = 6'],
+		onValidateSet(set) {
+			if (!set.item) return;
+			const item = this.dex.items.get(set.item);
+			if (item.itemUser && !this.ruleTable.has(`+item:${item.id}`)) {
+				return [`${set.species}'s item ${item.name} is banned.`];
+			}
+		},
+		onValidateRule() {
+			const table = this.ruleTable;
+			if (table.maxTeamSize < 6 || table.minTeamSize !== 6 || table.pickedTeamSize !== 6) {
```

Comment:
```suggestion
			if (table.pickedTeamSize < 6) {
```

Nothing else is actually relevant to this: bringing >6 pokemon will not crash this rule, and Picked Team Size already enforces team sizes.